srcpkgs/python3-pycodestyle/template

```diff
@@ -12,7 +12,8 @@ license="MIT"
 homepage="https://github.com/PyCQA/pycodestyle"
 changelog="https://raw.githubusercontent.com/PyCQA/pycodestyle/main/CHANGES.txt"
 distfiles="${PYPI_SITE}/p/pycodestyle/pycodestyle-${version}.tar.gz"
-checksum=347187bdb476329d98f695c213d7295a846d1152ff4fe9bacb8a9590b8ee7053
+checksum=259bcc17857d8a8b3b4a2327324b79e5f020a13c16074670f9c8c8f872ea76d0
+make_check=no # needs tox for testing
```

Comment:
this is incorrect, tox is not needed for testing, the issue here is only that the pypi tarball doesn't contain the testing module, fixed in 384390f0770dce0ef59b1c6959e11a146cda5591